issuing_using_did.md

```diff
@@ -0,0 +1,224 @@
+###### Authors:
```

Comment:
Why is identity verification optional ? (As described in this documentation correct me if I didn't understand correctly)

Also then why is identity proof required when using document store or token registry (with the current methods) ? We could displayed the document store address as well

Reply:
Identity verification is not optional... What i mean is that DID itself can be top level identifier displayed to user. It is optional to link that did to the DNS.

Reply:
The same line of thought, instead of using DNS, I can have another DID as my top level identifier.

Reply:
isnt did a bit too flex for identity ?
- did:ethr is unreadable and doesnt tell who you are
- did:github is readable but I can create one looking like another and pretend to be that person (raymond.yeh vs yeh.raymond)

maybe some did can act as true identity verifier (corppass ?) but I'm raising that because until now we were quite strict on identity: you must own a domain and proof you are the owner of the domain. While we can argue it could still be compromised by a malicious person, I think it's still a strong proof of identity ownership.

If we go the way of did is ownership then I'm scared that we become far less strict on the identity ownership and it becomes easier for malicious person to act as someone else (for instance by using did:ethr and using an address that looks like someone else address, just check few chars at the beginning and the end). We can also argue that "it's not our problem" but then it must be clear that users must be careful on that.

TLDR: I think this approach makes a 180° turn on how we consider identity on open attestation.

Reply:
We discussed about it and Raymond agreed on the above (feel free to correct me Raymond). I'm keeping this open to gather further feedbacks:
- even if it's a different approach is it a bad approach ?
- if we dont enforce dns resolution we can strongly advice to use it

Reply:
(Also we can enforce dns use from the product perspective, for instance directly in opencerts)